app/controllers/request_game_controller.rb

```diff
@@ -25,11 +25,13 @@ def play
               order('random()')
 
     if @missing == 0
-      flash[:notice] = {:partial => "game_over.html.erb",
-                        :locals => {
-                          :helpus_url => help_credits_path(:anchor => "helpus"),
-                          :site_name => site_name }
-                        }
+      flash.now[:notice] = {
+        :partial => "request_game/game_over.html.erb",
```

Comment:
won't hurt if there's any chance it will get confused - did break the tests tho 🤣